index.html

```diff
@@ -927,6 +927,9 @@
                 <li>If the pointer event dispatched was <code>pointerup</code> or <code>pointercancel</code>, clear the <i>PREVENT MOUSE EVENT</i> flag for this <code>pointerType</code>.</li>
             </ol>
             <div class="note">
+                <p>If the user agent supports both Touch Events (as defined in [[TOUCH-EVENTS]]) and Pointer Events, generation of compatibility mouse events as described in this section is not very web compatible. In this scenario gesture based generation of compatiblity mouse events for primary PointerEvents is better. For more information see [[TOUCH-EVENTS]].</p>
```

Comment:
"In this scenario gesture based generation of compatiblity mouse events for primary PointerEvents is better." do we actually define what "gesture-based generation" (I'd add a hyphen) is anywhere?